library/src/scala/quoted/Liftable.scala

```diff
@@ -13,6 +13,23 @@ abstract class Liftable[T] {
  *  gives an alternative implementation using just the basic staging system.
  */
 object Liftable {
-  implicit def IntIsLiftable: Liftable[Int] = ???
-  implicit def BooleanIsLiftable: Liftable[Boolean] = ???
+
+  sealed abstract class PrimitiveExpr[T] extends Expr[T] {
+    def value: T
+  }
+
+  private class ValueExpr[T <: AnyVal](val value: T) extends PrimitiveExpr[T]
```

Comment:
Doesn't look like we need `ValueExpr` as a separate class.

Reply:
I will remove it